opflexagent/as_metadata_manager.py

```diff
@@ -82,6 +85,22 @@
 STATE_FILENAME_NETS = STATE_FILE_NAME_FORMAT % STATE_INSTANCE_NETWORKS
 
 
+def get_bridge_manager(conf):
```

Comment:
We should move this into a shared utils directory, and have the gbp_agent use the same method. Maybe create a new opflexagent/utils/utils.py file for this?

Reply:
Accepted. Have changed this